### ISSUE TITLE
Recreate Gif in Desktop Course: review-local-workflow.gif

### ISSUE THREAD

**Issue by brianamarie** [github/training-kit#513]
## Overview
**TL;DR**
The current [on-demand desktop course](https://services.github.com/on-demand/github-desktop/) features a .gif that isn't accurate with the name of the file. 

### Extra Detail
Right now, the file _should_ be named `githubstudent.yaml` but is named `githubteachera.yaml`. This .gif should be recreated for accuracy. 

cc/ @github/services-training 

#### Screenshots
![](https://github.com/github/training-kit/blob/master/images/gifs/github-desktop/review-local-workflow.gif)

### How to Contribute
- [ ] Comment on this issue if you'd like to work/collaborate on it! 
- [ ] Fork the [repository](https://github.com/github/training-kit).
- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. [This](https://services.github.com/on-demand/intro-to-github/) is a refresher course if you're unsure about how to make a change on GitHub.
- [ ] Push your changes to your repository. 
- [ ] Submit a Pull Request
  - Base Dropdown: github/training-kit
  - Compare Dropdown: Your fork

#### Testing Locally
If you'd like to make and test changes locally (and see how they would look if merged), do the following:
- In your command line:
  - [Install Git](https://git-scm.com/)
  - [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
  - [ ] Run `script/setup`
  - [ ] Run `script/server`.
    - When successful, the script will initiate a local server at `http://127.0.0.1:4000/on-demand`.

### Questions?
- Leave a comment on this issue! Make sure to use @ mentions if you want a specific person's attention!
